tests/test_flows.py

```diff
@@ -80,6 +80,7 @@ def check_cmdu(
         result = env.wired_sniffer.get_cmdu_capture(match_function)
         if not result:
             self.fail("No CMDU {} found".format(msg))
+            assert False
```

Comment:
(nitpick)
```suggestion
        assert result, "No CMDU {} found".format(msg)
```

You already had something like this below, so I guess it was just an oversight :-) But there are still other cases that could be simplified like this.